SocketRocket/Internal/Security/SRPinningSecurityPolicy.h

```diff
@@ -13,6 +13,9 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+// NOTE: While publicly, SocketRocket does not support configuring the security policy with pinned certificates,
```

Comment:
Nit: Use a section comment aka `/** ... */` as well as document the class itself?